OpenOversight/migrations/versions/4a490771dda1_add_original_image_fk.py

```diff
@@ -0,0 +1,34 @@
+"""add_original_image_fk
+
+Revision ID: 4a490771dda1
+Revises: bd0398fe4aab
+Create Date: 2018-06-07 15:32:25.524117
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '4a490771dda1'
+down_revision = '8ce7926aa132'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('faces', sa.Column('fk_face_original_image_id', sa.Integer(), nullable=True))
+    op.drop_constraint(u'faces_img_id_fkey', 'faces', type_='foreignkey')
+    op.create_foreign_key('fk_face_image_id', 'faces', 'raw_images', ['img_id'], ['id'], onupdate='CASCADE', ondelete='CASCADE', use_alter=True)
+    op.create_foreign_key(None, 'faces', 'raw_images', ['fk_face_original_image_id'], ['id'], onupdate='CASCADE', ondelete='SET NULL', use_alter=True)
+    # ### end Alembic commands ###
```

Comment:
Verified tags (mapping of face -> officer) are indeed preserved after this migration ✅